examples/java/com/example/JavaNativeFuzzTest.cpp

```diff
@@ -16,15 +16,16 @@
 
 #include <string>
 
-// simple function containing a crash that requires coverage and string compare
-// instrumentation for the fuzzer to find
+// Prevent the compiler from optimizing away the out-of-bounds read in
+// parseInternal.
+volatile int dummy = 0;
+
```

Comment:
Maybe we can achieve the same result by marking the function below as `__attribute__((optnone))`?

Reply:
Thanks, I didn't know this attribute.

Reply:
Thanks, I didn't know this attribute.